src/main/java/Ken/ui/Ui.java

```diff
@@ -0,0 +1,24 @@
+package ken.ui;
+
+/**
+ * The Ui class is responsible for handling user interface-related messages.
+ * It provides methods to display welcome and goodbye messages to the user.
+ */
+public class Ui {

```

Comment:
If you already have a Ui class, would it be better to shift all `System.out::println` from other classes to here?